.gitignore

```diff
@@ -125,8 +125,14 @@ docs_zh-CN/topics/
 docs_zh-CN/papers/*.md
 docs_zh-CN/datasets.md
 docs_zh-CN/modelzoo.md
-
+mmcv/
```

Comment:
The modification of this file may be not necessary.

Reply:
Sure. You can simply delete them.